manticore/__main__.py

```diff
@@ -4,13 +4,54 @@
 import argparse
 
 from . import Manticore
-from .utils import log
+from .utils import log, config
 
 # XXX(yan): This would normally be __name__, but then logger output will be pre-
 # pended by 'm.__main__: ', which is not very pleasing. hard-coding to 'main'
 logger = logging.getLogger('manticore.main')
 
-sys.setrecursionlimit(10000)
+consts = config.get_group('main')
+consts.add('recursionlimit', default=10000,
+           description="Value to set for Python recursion limit")
+
+sys.setrecursionlimit(consts.recursionlimit)
+
+
+def process_config_values(args, parser):
```

Comment:
nit, i'm a little wary of adding stuff to this file :/ i just don't want it to get really big since it's supposed to just be a small thing and has been growing

Reply:
Maybe this should go to config.py  (or helpers?)